.github/dependabot.yml

```diff
@@ -1,66 +1,49 @@
 ---
 version: 2
 updates:
-  - directory: /
-    # ignore:
-    #   # Managed by cisagov/ASM-Dashboard
-    #   - dependency-name: actions/cache
-    #   - dependency-name: actions/checkout
-    #   - dependency-name: actions/setup-go
-    #   - dependency-name: actions/setup-python
-    #   - dependency-name: crazy-max/ghaction-dump-context
-    #   - dependency-name: crazy-max/ghaction-github-labeler
-    #   - dependency-name: crazy-max/ghaction-github-status
-    #   - dependency-name: hashicorp/setup-terraform
-    #   - dependency-name: mxschmitt/action-tmate
-    #   - dependency-name: step-security/harden-runner
-    package-ecosystem: github-actions
```

Comment:
Any upstream dependencies should be handled by the parent repo (e.g. skeleton-generic, in this case).  For an example of how we typically do that, please see [here](https://github.com/cisagov/pshtt/blob/17d7598993053d30e03455d30baf03b77392d74e/.github/dependabot.yml#L3-L30).

Reply:
This will look a bit different because I refactored to match the existing dependabot.yml code from cisagov/crossfeed.

Reply:
For the future, we let the person who opened the conversation be the one to resolve it.  Also, it's helpful to include the commit where you addressed the concern in the convo (in this case it was https://github.com/cisagov/XFD/pull/32/commits/adb7284c9742ebd1ed18c7274b2b434e0daf2310).  This looks good to me, so we can leave this convo as resolved.